cvat/apps/engine/permissions.py

```diff
@@ -456,7 +462,7 @@ def create(cls, request, view, obj, iam_context):
     def create_scope_view(cls, request, task: Union[int, Task], iam_context=None):
         if isinstance(task, int):
             try:
-                task = Task.objects.get(id=task)
+                task = Task.objects.select_related("organization").get(id=task)
```

Comment:
Could you explain this? It seems to go against the purpose of this PR, as it adds a _new_ join.

Reply:
Did you mean `select_related("project")`?